packages/backend/src/api-impl.ts

```diff
@@ -70,6 +72,28 @@ export class BootcApiImpl implements BootcApi {
     return buildDiskImage(build, this.history, overwrite);
   }
 
+  // Launches a macadam VM by initializing the class and initializing the VM
+  async createVM(options: CreateVmOptions): Promise<void> {
+    const macadam = new MacadamVirtualMachine();
+    try {
+      await macadam.init();
```

Comment:
Is there any downside to creating a new MacadamVirtualMachine and initializing it every time? Or conversely does it get out to sync if you create it once and reuse later?

e.g. should we have a getMacadamVirtualMachine function that creates and inits it once, then returns the same instance the next time it is called? (doing this as follow-up is totally fine too)

Reply:
After reviewing MacadamVirtualMachine I'm leaning more toward init() being an implementation details, i.e. line 79 should be removed and the class automatically calls init when you create it, or the first time you call createVM or listVM. (but I'm still wondering if we should only create one shared instance of MacadamVirtualMachine here)

Reply:
I've moved init() into the createVm and listVms.

I tried putting it into the constructor, but with our typescript lint checking, we do not allow it:
```
/Users/cdrage/syncthing/dev/electron/podman-desktop-extension-bootc/packages/backend/src/macadam.ts
  36:5  error  Refactor this asynchronous operation outside of the constructor  sonarjs/no-async-constructor
```

But I've at least moved init() into the functions so that we do not have to call init() each time in `api-impl.ts`.